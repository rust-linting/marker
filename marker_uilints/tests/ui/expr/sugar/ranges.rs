fn main() {
    #[warn(marker::marker_uilints::print_every_expr)]
    let _ = 1..2;
    #[warn(marker::marker_uilints::print_every_expr)]
    let _ = ..2;
    #[warn(marker::marker_uilints::print_every_expr)]
    let _ = 1..;
    #[warn(marker::marker_uilints::print_every_expr)]
    let _ = ..;
}
