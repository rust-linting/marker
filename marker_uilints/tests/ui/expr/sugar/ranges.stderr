warning: expr
 --> $DIR/ranges.rs:3:13
  |
3 |     let _ = 1..2;
  |             ^^^^
  |
  = note: SpanSource: File(
              FileInfo {
                  file: "$DIR/ranges.rs",
                  span_src: SpanSrcId(..),
              },
          )
  = note: Snippet: "1..2"
note: the lint level is defined here
 --> $DIR/ranges.rs:2:12
  |
2 |     #[warn(marker::marker_uilints::print_every_expr)]
  |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

warning: expr
 --> $DIR/ranges.rs:3:13
  |
3 |     let _ = 1..2;
  |             ^
  |
  = note: SpanSource: File(
              FileInfo {
                  file: "$DIR/ranges.rs",
                  span_src: SpanSrcId(..),
              },
          )
  = note: Snippet: "1"

warning: expr
 --> $DIR/ranges.rs:3:16
  |
3 |     let _ = 1..2;
  |                ^
  |
  = note: SpanSource: File(
              FileInfo {
                  file: "$DIR/ranges.rs",
                  span_src: SpanSrcId(..),
              },
          )
  = note: Snippet: "2"

warning: expr
 --> $DIR/ranges.rs:5:13
  |
5 |     let _ = ..2;
  |             ^^^
  |
  = note: SpanSource: File(
              FileInfo {
                  file: "$DIR/ranges.rs",
                  span_src: SpanSrcId(..),
              },
          )
  = note: Snippet: "..2"
note: the lint level is defined here
 --> $DIR/ranges.rs:4:12
  |
4 |     #[warn(marker::marker_uilints::print_every_expr)]
  |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

warning: expr
 --> $DIR/ranges.rs:5:15
  |
5 |     let _ = ..2;
  |               ^
  |
  = note: SpanSource: File(
              FileInfo {
                  file: "$DIR/ranges.rs",
                  span_src: SpanSrcId(..),
              },
          )
  = note: Snippet: "2"

warning: expr
 --> $DIR/ranges.rs:7:13
  |
7 |     let _ = 1..;
  |             ^^^
  |
  = note: SpanSource: File(
              FileInfo {
                  file: "$DIR/ranges.rs",
                  span_src: SpanSrcId(..),
              },
          )
  = note: Snippet: "1.."
note: the lint level is defined here
 --> $DIR/ranges.rs:6:12
  |
6 |     #[warn(marker::marker_uilints::print_every_expr)]
  |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

warning: expr
 --> $DIR/ranges.rs:7:13
  |
7 |     let _ = 1..;
  |             ^
  |
  = note: SpanSource: File(
              FileInfo {
                  file: "$DIR/ranges.rs",
                  span_src: SpanSrcId(..),
              },
          )
  = note: Snippet: "1"

warning: expr
 --> $DIR/ranges.rs:9:13
  |
9 |     let _ = ..;
  |             ^^
  |
  = note: SpanSource: File(
              FileInfo {
                  file: "$DIR/ranges.rs",
                  span_src: SpanSrcId(..),
              },
          )
  = note: Snippet: ".."
note: the lint level is defined here
 --> $DIR/ranges.rs:8:12
  |
8 |     #[warn(marker::marker_uilints::print_every_expr)]
  |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

warning: 8 warnings emitted

